File: C#/2748 - Saída 2.cs
/// author Josélio Júnior (Lunatic Fox) <joseliojrx25@gmail.com>
/// copyright Josélio Júnior (Lunatic Fox) 2022

using System;
using System.Linq;

class URI {
  static void Main(string[] args) { 
    string b = new string(' ', 39).Replace(" ", "-");
    string i = $"|{new string(' ', 37)}|\n";
    string[] w = new string[]{ "Roberto", "5786", "UNIFEI" }
      .Select(e => {
        string ve = $"|{new string(' ', 8)}{e}";
        return $"{ve}{new string(' ', 38 - ve.Length)}|\n";
      })
      .ToArray();

    Console.WriteLine($"{b}\n{string.Join(i, w)}{b}");
  }
}

File: Lua/2748 - Saída 2.lua
--- author: Josélio Júnior (Lunatic Fox) <joseliojrx25@gmail.com>
--- copyright: Josélio Júnior (Lunatic Fox) 2022

local w = { 'Roberto', '5786', 'UNIFEI' }
local function l(x) return ('|%s%s%s|\n'):format((' '):rep(8), x, (' '):rep(29 - #x)) end
local b, s = ('-'):rep(39), ('|%s|\n'):format((' '):rep(37))
print(('%s\n%s%s%s%s%s%s'):format(b, l(w[1]), s, l(w[2]), s, l(w[3]), b))